src/h3d/col/ConvexHull.hx

```diff
@@ -0,0 +1,341 @@
+package h3d.col;
+
+//import h3d.Vector;
+
+
+class ConvexHull {
+	static final VALIDATE = true;
+
+	// Fundamental Data -------
+
+	// Base list of points, most references will be via indices
+	public var points: Array<Vector>;
+	// faces is a flat array of 3 int tuples, indices into points
+	// A valid face list must be closed and have all faces oriented facing
+	// outward righthanded
+	// Conveniently, this is also basically the index array for rendering triangles
+	public var faces: Array<Int>;
+
+	// Derived Data --------
+	// Everything below can be derived from points and faces above
+
+	// A flat 3 tuple for each face of the indices of the connected faces
+	public var faceConnections: Array<Int> = new Array();
+
+	public var facePlanes: Array<Plane> = new Array();
+
+	// A list per point of connected points
+	public var pointConnections: Array<Array<Int>> = new Array();
```

Comment:
If it says points, it's one-to-one with points.

If it says faces it's one-to-one with faces. Or one-to-three since I flattened the array-of-arrays. It depends on context.

Reply:
Having a duh moment as it says per. :1st_place_medal: